deploy/templates/auth/principals/service-clients.yaml

```diff
@@ -122,3 +122,47 @@ spec:
       group: {{ .Values.acs.organisation }}-Service-Core
       node: Monitor
 {{- end }}
+---
+{{- if .Values.unsIngesters.sparkplug.enabled }}
+apiVersion: factoryplus.app.amrc.co.uk/v1
+kind: KerberosKey
+metadata:
+  name: sv1sparkplugingester
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: Random
+  principal: sv1sparkplugingester@{{ .Values.identity.realm | required "values.identity.realm is required!" }}
+  secret: uns-ingester-sparkplug-keytabs/client
+  account:
+    class: e463b4ae-a322-46cc-8976-4ba76838e908
+    name: Sparkplug Ingester
+    groups:
+      - e414d355-b991-429b-8f5d-97e823ff71f5
+    sparkplug:
+      group: {{ .Values.acs.organisation }}-Service-Core
+      node: SparkplugIngester
+
+{{- end }}
+---
+# In the future if we have any more ingesters then this should be
+# enabled if any of the ingesters are enabled.
+{{- if .Values.unsIngesters.sparkplug.enabled }}
+apiVersion: factoryplus.app.amrc.co.uk/v1
+kind: KerberosKey
+metadata:
+  name: sv1historianuns
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: Random
+  principal: sv1historianuns@{{ .Values.identity.realm | required "values.identity.realm is required!" }}
+  secret: historian-uns-keytabs/client
+  account:
+    class: e463b4ae-a322-46cc-8976-4ba76838e908
+    name: Historian UNS
+    groups:
+      - 03f5f08a-f61e-4134-8f66-b2951e3bbb69
+    sparkplug:
+      group: {{ .Values.acs.organisation }}-Service-Core
+      node: HistorianUNS
```

Comment:
A historian also probably doesn't publish (unless it's reporting on its own status?)